pysat/tests/test_params.py

```diff
@@ -27,20 +28,32 @@ def setup(self):
         # Set up default values
         pysat.params.restore_defaults()
 
+        # Get a temporary directory
+        self.tempdir = tempfile.TemporaryDirectory()
+        self.wd = os.getcwd()
+
     def teardown(self):
         """Runs after every method to clean up previous testing."""
         pysat.params = copy.deepcopy(self.stored_params)
         pysat.params.store()
         reload(pysat)
+        self.tempdir.cleanup()
+        os.chdir(self.wd)
 
     @pytest.mark.parametrize("paths, check",
                              [('.', ['.']),
+                              ('./hi', ['./hi']),
+                              ('./hi/', ['./hi']),
+                              ('./', ['.']),
```

Comment:
These tests will fail on windows.  recommend using `os.paths` to construct the test directories.